qutebrowser/config/configtypes.py

```diff
@@ -1336,6 +1336,21 @@ def __init__(self, none_ok=False):
             valid_values=ValidValues('top', 'bottom', 'left', 'right'))
 
 
+class TitleAlignment(MappingType):
+
+    """Alignment of tab text"""
```

Comment:
I'd call this `TextAlignment` and change the docstring accordingly - I think this makes sense as a generic type and could be used for other config options in the future too.